lanternmq/definitions.json

```diff
@@ -8,6 +8,12 @@
             "hashing_algorithm": "rabbit_password_hashing_sha256",
             "tags": ""
         },
+        {
```

Comment:
Looks like this file was re-modified to have the netstats queue information

Reply:
I must have made a mistake while rebasing, thanks for catching this! It is all set now